ExadelTimeTrackingSystem/ExadelTimeTrackingSystem.BusinessLogic/DTOs/CreateUserDTO.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace ExadelTimeTrackingSystem.BusinessLogic.DTOs
+{
+    using ExadelTimeTrackingSystem.Data.Models.Enums;
+
+    public class CreateUserDTO
+    {
+        public string FullName { get; set; }
+
+        public string Email { get; set; }
+
+        public string Username { get; set; }
+
+        public string Password { get; set; }
+
+        public Role[] Roles { get; set; }
```

Comment:
use enum from dtos

Reply:
and please create UserProfile and register mappings, including enum mappings, same for TaskProfile, and register profiles similar to ProjectProfile